src/components/Tooltip.vue

```diff
@@ -34,217 +34,363 @@ import { Component, Prop, Vue, Watch } from 'vue-property-decorator';
 import { AlertTriangleIcon } from './Icons';
 
 @Component({ components: { AlertTriangleIcon }})
-export default class Tooltip extends Vue {
-    // Only $el of the reference is of interest
-    @Prop(Object) public reference?: {$el: HTMLElement};
+class Tooltip extends Vue {
+    @Prop(Object) public container?: Vue | {$el: HTMLElement};
+    @Prop(Boolean) public disabled?: boolean;
+
+    @Prop({
+        type: String,
+        default: 'bottom right',
```

Comment:
I think the default should be 'top center'. Or 'top right' if Center is not available.

Tooltips usually open above the trigger usually, I believe.

Reply:
I, too, would say that they usually are either top center, top right, or right center. However if in our specific use case the position. is predominantly bottom right, and it is anticipated to stay that way I am fine with bottom right, too.

Reply:
In my opinion this is not about which direction is used most often in _our_ apps.

We are publishing the vue-components not just for our usage, but also for the communtiy. _We_ know about the `preferredPosition` prop, but a dev who uses the tooltip component for the first time might not. And when using a tooltip component (without props), I would expect it to be `top right`. When I don't like this default, I can look up the options to change it.